### ISSUE TITLE
Scene editor: Adding items causes NullReferenceExceptions

### ISSUE THREAD

**Issue by tonikone** [mdotstrange/NightmarePuppeteerPublic#162]
Occured in the Forest. Tried adding an item annotated in the picture:

https://imgur.com/a/fxzOUNH

Log:
[output_log_adding_items_forest.txt](https://github.com/mdotstrange/NightmarePuppeteerPublic/files/5154198/output_log_adding_items_forest.txt)

There is a "Failed to create agent because it is not close enough to the NavMesh". Is this an actor placement error or something else?

**Comment by mdotstrange** [comment]
The error on this line which is the null ref   at NpSceneEditor.DoubleSizeOfObject () [0x0001f] in H:\CreativeGit\Puppets\Assets\Scripts\NpLevelEditor\NpSceneEditor.cs:364 

That is from hitting the x2 button when there is no selected object- added a null check on that one-

The failed to create navmesh isn't an error- just a warning

**Comment by tonikone** [comment]
Yes, x2 was used on a small object and in haste. Some sort of item scale and rotation normalization against general actor proportions would be nice. Per object asset tinkering, I'm guessing. Can the Editor models be packed into Asset Bundles that supports your workflow? I could take a look.

**Comment by tonikone** [comment]
The scaling issue touches Actor props only. It would be nice to have good human scale proportions rotations for those little cars, trucks etc. Some sort of default normalization in the early pipeline would be good in the long run.

**Comment by mdotstrange** [comment]
Yeah having human scale proportions would make sense for the editor- right now those are the same props that are being used by the actors so they are scaled for that- I'll look into adjusting their scales at runtime depending on use-
